### DISCUSSION TITLE
[x][closed] Daniel: I don't think we can require the use of target if the definition is not nailed to the floor. [Experiment]

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#91] [Team Posts]
Daniel: IHEC is also struggling with this issue, so I guess this will take a while to resolve

**Comment by sveinugu** [comment]
@dzerbino I somewhat agree, but on the other hand I fell that target is perhaps (together with sample_type) the most important metadata attribute. We could settle it by providing a term_iri for "not applicable" or similar as a default value.

**Comment by dzerbino** [comment]
Current IHEC thinking:
- TF: HGNC ID + free text for possible Post-Translational Modifications (PTM)
- Chromatin: controlled vocab

**Comment by sveinugu** [comment]
Closing this, as the discussion continues in https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/19
